frontend/src/components/editor/ai/add-cell-with-ai.tsx

```diff
@@ -450,3 +464,13 @@ export const PromptInput = ({
     />
   );
 };
+
+/** Converts a prompt to a markdown cell. Used to display the prompt in the UI. */
+export function prettifyPromptToMarkdown(prompt: string): string {
+  const footer = '<p align="center"><i>Generating with AI...</i></p>';
+  const wrappedPrompt = `\`\`\`md\n${prompt}\n\`\`\`\n${footer}`;
+  const [markdownCode] = LanguageAdapters.markdown.transformOut(wrappedPrompt, {
+    quotePrefix: "f",
+  });
+  return markdownCode;
```

Comment:
i think just the prompt and no styling is cleaner. something that they can choose to commit to the notebook should they want some history of the prompt.


in future, we may want to add some hidden metadata that _could_ be use to regenerate the cells below